resources/js/features/game-list/utils/column-definitions/buildHasActiveOrInReviewClaimsColumnDef.tsx

```diff
@@ -0,0 +1,55 @@
+import type { ColumnDef } from '@tanstack/react-table';
+import type { RouteName } from 'ziggy-js';
+
+import {
+  BaseTooltip,
+  BaseTooltipContent,
+  BaseTooltipTrigger,
+} from '@/common/components/+vendor/BaseTooltip';
+
+import { DataTableColumnHeader } from '../../components/DataTableColumnHeader';
+
+interface BuildHasActiveOrInReviewClaimsColumnDefProps {
+  tableApiRouteName?: RouteName;
+}
+
+export function buildHasActiveOrInReviewClaimsColumnDef({
+  tableApiRouteName = 'api.game.index',
+}: BuildHasActiveOrInReviewClaimsColumnDefProps): ColumnDef<App.Platform.Data.GameListEntry> {
+  return {
+    id: 'hasActiveOrInReviewClaims',
+    accessorKey: 'game',
+    meta: { label: 'Claimed', align: 'right' },
```

Comment:
`align: 'right'` may not be right. Maybe it'd be best if this is left-aligned?

Reply:
I agree. left probably makes more sense here

Reply:
👍  Left-aligned in latest.